clients/java/src/test/java/io/camunda/zeebe/client/util/RestGatewayService.java

```diff
@@ -69,6 +70,14 @@ public void onTopologyRequest(final TopologyResponse topologyResponse) {
                 .willReturn(WireMock.okJson(JSON_MAPPER.toJson(topologyResponse))));
   }
 
+  public void onEvaluateDecisionRequest(final EvaluateDecisionResponse response) {
+    mockInfo
+        .getWireMock()
+        .register(
+            WireMock.get(RestGatewayPaths.getEvaluateDecisionUrl())
```

Comment:
🔧 @nicpuppa you need to use a `post` here. That's why two of the `DecisionEvaluationRestTest` test cases are failing:

```suggestion
            WireMock.post(RestGatewayPaths.getEvaluateDecisionUrl())
```